test/resources/device_config.yml

```diff
@@ -21,6 +21,9 @@ DEVICES:
     DEVICE_CLASS: moler.device.unixlocal.UnixLocal
     INITIAL_STATE: UNIX_LOCAL
 
+  UNIX_LOCAL_CLONED_VIA_YAML:
+    CLONED: UNIX_LOCAL
```

Comment:
I would change name to `CLONE_OF` or `CLONED_FROM`. Current naming is little bit confusing. You may read it as: **UNIX_LOCAL_CLONED_VIA_YAML has cloned UNIX_LOCAL (and what? stored it somewhere? thrown it away?)**

However, sentences
* UNIX_LOCAL_CLONED_VIA_YAML is `clone of` UNIX_LOCAL or 
* UNIX_LOCAL_CLONED_VIA_YAML has been `cloned from` UNIX_LOCAL
are self explanatory.

Reply:
Changed into **CLONED_FROM**